src/go/k8s/controllers/redpanda/redpanda_controller.go

```diff
@@ -85,6 +85,31 @@ type RedpandaReconciler struct {
 // +kubebuilder:rbac:groups="monitoring.coreos.com",namespace=default,resources=servicemonitors,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=networking.k8s.io,namespace=default,resources=ingresses,verbs=get;list;watch;create;update;patch;delete
 
+// additional k8s resources required by flux (Cluster Scope)
+// The redpanda chart requires, for rack-awareness, that we have access to cluster scope resources.
+// So for a redpanda resource to enable rack-awareness the operator requires permissions
+// at the cluster scope.
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles,verbs=get;list;watch;create;update;patch;delete
+
+// additional K8s resources (Cluster Scope) defined in rbac.yaml for the redpanda chart (rpk-bundle)
+// The redpanda chart requires, for rack-awareness, that we have access to cluster scope resources.
+// So for a redpanda resource to enable rack-awareness the operator requires permissions
+// at the cluster scope.
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=endpoints,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=events,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=limitranges,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=persistentvolumeclaims,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=pods/log,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=replicationcontrollers,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=resourcequotas,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=services,verbs=get;list;
+// +kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;
+// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Where rpk bundle needs all permissions for custom resource definitions?

Reply:
The rack-awareness feature doesn't need custom resource definition create, patch, update nor delete verbs.

We have dedicated step to install all required CRDs
https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/kubernetes/kubernetes-deploy/?tab=tabs-1-helm-operator#deploy-redpanda
> Install the Redpanda Operator CRDs:

https://github.com/redpanda-data/docs/blob/7952595222a74b041ac0ab1c1f9f4bf5618e700f/modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-deploy.adoc?plain=1#L46-L50

In the rpk debug bundle 
https://github.com/redpanda-data/redpanda/blob/c8262bf0c3b5084b45ae1b6a404e5e3583614b1f/src/go/rpk/pkg/cli/debug/bundle/bundle_k8s_linux.go#L328-L339

We don't read nor list the custom resource definitions. 

That's why I'm confused with the statement 
> this is an rpk requirement, and this is required for us to deploy redpanda helm charts